api/include/mfxastructures.h

```diff
@@ -119,7 +120,13 @@ typedef struct {
         };
     };
 } mfxAudioInfoMFX;
+#pragma pack(pop)
 
+#if defined(_WIN64) || defined(LINUX64)
```

Comment:
Wait!!! This seems to be a direct regression to mediasdk end-user experience who will require to set compiler undefined LINUX32 and/or LINUX64 properly. (I thinkw _WIN32 and _WIN64 are compiler-defined, right?)

We need to think about some other approach.

Reply:
I've changed approach to get rid of ```LINUX32```/```LINUX64``` macro. Yes, ```_WIN32```/```_WIN64``` are compiler defined